packages/wdio-allure-reporter/tests/suite.test.js

```diff
@@ -1,6 +1,9 @@
 import { directory } from 'tempy'
+
+// eslint-disable-next-line
+import { clean, getResults } from 'wdio-allure-helper'
```

Comment:
seems unrelated changes

Reply:
ah good catch

Reply:
actually it an intended change because I refactored the way the helper is used and abuse the package mocking. Added a comment for clarification.